elements/rh-spinner/rh-spinner.ts

```diff
@@ -59,6 +43,23 @@ export class RhSpinner extends BaseSpinner {
       <slot></slot>
     `;
   }
+
+  // START hack for removal of contextProvider. delete for version 2.0
+  /**
+   * Deprecated: Use Color context instead
```

Comment:
Should be a link that explains what this means

Reply:
we don't have a page that explains this yet :/

Reply:
@marionnegp @brianferry until we write a page that explains the context system, can we just merge this?

Reply:
Can we include more info in the README for now?